lib/assets/stylesheets/nyulibraries/_misc.scss

```diff
@@ -15,11 +15,16 @@
   min-width: 200px;
 }
 
+.btn1 {
+  line-height: $line-height-large-computed;
+}
+
```

Comment:
@Amay22 I think this works the original way. If you `@extend .btn` then you are getting all the bootstrap btn styles (https://github.com/twbs/bootstrap-sass/blob/d35d4bd58708c3db82aaac7b50295ff37a2d1c83/assets/stylesheets/bootstrap/_buttons.scss#L9) but changing the line-height. 

Can you explain what your changes are doing differently?